tests/test_observations.py

```diff
@@ -136,3 +138,86 @@ def test_tomo_nz_gausserr() -> None:
     # check the shape of the output
 
     np.testing.assert_array_equal(binned_nz.shape, (len(zbins), len(z)))
+
+
+@pytest.mark.parametrize(
```

Comment:
I started to add tests yesterday before @ntessore's review. It feels pointless continuing these given the proposed changes.